Sources/SwiftBasicFormat/BasicFormat.swift

```diff
@@ -508,6 +508,17 @@ open class BasicFormat: SyntaxRewriter {
       return token
     }
 
+    if token.text == "\n" && token.leadingTrivia.isEmpty && token.trailingTrivia.isEmpty {
+      return token
+    }
+    if token.text.isEmpty
+        && token.leadingTrivia.isEmpty
+        && token.trailingTrivia.startsWithNewline
+        && token.trailingTrivia.endsWithNewline
+        && previousTokenWillEndWithNewline {
+      return token
+    }
```

Comment:
I noticed some minor formatting differences in this code compared to other code in the repo. Would you mind [runing `swift-format`](https://github.com/apple/swift-syntax/blob/main/CONTRIBUTING.md#formatting)?